cmd/queue/main.go

```diff
@@ -214,8 +215,57 @@ func createAdminHandlers() *http.ServeMux {
 	return mux
 }
 
+func execHTTPAction(url string) error {
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusBadRequest {
+		return fmt.Errorf("HTTP request failed with status: %v", resp.StatusCode)
+	}
+
+	return nil
+}
+
+func buildAdminUrl(path string) string {
+	url :=  url.URL{
+		Scheme: "http",
+		Host:   fmt.Sprintf("localhost:%d", v1alpha1.RequestQueueAdminPort),
+		Path:   path,
+	}
+
+	return url.String()
+}
+
+func handleActions() (bool, error) {
+	var err error
+
+	switch flag.Arg(0) {
+	case queue.CommandModeExec:
+		if len(flag.Args()) > 1 {
+			actionMap := map[string]string{
```

Comment:
This is great, thanks.
Now the last thing, can you move it outside of function? I understand this is executed only once so there's no performance gain, but the flow of the code might be better.

Reply:
That's fair, I'll update that.